src/lib.rs

```diff
@@ -46,6 +46,9 @@
 #![allow(clippy::identity_op)]
 #![allow(clippy::unreadable_literal)]
 
+#[cfg(all(feature = "openssl", feature = "no_openssl"))]
+compile_error!("feature \"openssl\" and feature \"no_openssl\" cannot be enabled at the same time");
```

Comment:
Would there be a scenario were we want the ability to have both enabled? @larrydewey Or do we best keep them separate?